### PR TITLE
Add usage example to pad_to_bounding_box

### PR DESCRIPTION
**Pull request by rickwierenga**

### CODE REVIEW

**Review by Mbah-Javis — CHANGES_REQUESTED** [review]
Great work! Just make this little changes. Change (1.0) to (1.) just by removing the 0 and do the same for other values also remove the spacing in-between the output

**Comment by rickwierenga** [comment]
I removed the 0's. Thanks for the tip. I actually think keeping the output is better because I copied it directly from the terminal so it's the output same other users will get. If you still think it's better to remove it, please say so and I'll do it.

**Review by Mbah-Javis — COMMENTED** [review]
No don't remove the output keep it but remove the empty lines in between the output array

**Comment by rickwierenga** [comment]
done.

**Comment by Mbah-Javis** [comment]
Good 👍

**Review by boronhub — COMMENTED** [review]
Hi there ! I have suggested a few changes, kindly go through them.

**Comment by rickwierenga** [comment]
Thanks!

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -849,6 +849,15 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
...
   `target_height` by `target_width`.
+  
+  Usage Example:
+    >>> x = [[[1., 2., 3.],
+    ...       [4., 5., 6.]],
+    ...       [[7., 8., 9.],
+    ...       [10., 11., 12.]]]
+    >>> padded_image = tf.image.pad_to_bounding_box(x, 1, 1, 4, 4)
+    >>> padded_image
+    <tf.Tensor: shape=(4, 4, 3), dtype=float32, numpy=array([...], dtype=float32)>
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py]: We would prefer the numbers to be visible in the numpy array

Comment (rickwierenga) [reply tensorflow/python/ops/image_ops_impl.py]: @boronhub said [...] syntax is better: https://github.com/tensorflow/tensorflow/pull/36056#discussion_r368853617. Should I add the numbers again?

Comment (mihaimaruseac) [reply tensorflow/python/ops/image_ops_impl.py]: Yes please. If the numbers are deterministic they should show up. If they are not, the test should be rewritten to make them so. Only if the test cannot be rewritten in that way should you go for `...`

Comment (rickwierenga) [reply tensorflow/python/ops/image_ops_impl.py]: All right.

**Comment by rickwierenga** [comment]
@mihaimaruseac is there anything else that needs to be changed?

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**